NewHorizons/Handlers/PlanetDestructionHandler.cs

```diff
@@ -40,7 +40,14 @@ public static void RemoveStockPlanets()
             {
                 foreach (var gameObject in toDisable)
                 {
-                    gameObject.SetActive(false);
+                    // Somehow SetActive can NRE
+                    // Seems to only happen if they don't have the DLC installed
+                    // Even with the null check it's complaining so I don't understand at all
+                    try
+                    {
+                        gameObject?.SetActive(false);
```

Comment:
? doesnt cover destroy. have to do == null or != null or bool check. some dlc stuff gets destroyed on start so we get it then and then 2 frames later its destroyed and we try to do setactive and it false

tdlr do != null guy

Reply:
swore I tried that, ok